src/middleware/bodyParser/formData.ts

```diff
@@ -0,0 +1,104 @@
+/*!
```

Comment:
This is your own work afaik? If so, don’t need to attribute copyright to another library so this notice can go.

Reply:
Well yeah, mostly. I'm not entierly sure if I still have to have the copyright there because I did copy from json.ts to formData.ts and just changed stuff around 🤷🏽  Gonna get rid of it for now since I actually wrote the rest of the middleware myself